### PR TITLE
Migrate "Using metadata" from Sanity to MDX

### LINKED ISSUE #253: Migration of metadata page from Sanity CMS to MDX
## Summary
I would like to help migrate the following clerk doc page from Sanity CMS to MDX: [Metadata](https://clerk.com/docs/organizations/metadata)

## Acceptance criteria
- [ ] Ensure all the content and links are the same and functional
- [ ] Use proper MDX components, themes, and headers for content, and headings

## Notes
Discussed with @alexisintech working on the above page

### PR DESCRIPTION
**Pull request by yiremorlans**
## Description
These changes include the transfer of all the content from the "Using metadata" docs to mdx.

## Related Issue
close #253 

## Acceptance Criteria
- [x] Ensure all the content and links are the same and functional
- [x] Use proper MDX components, themes, and headers for content, and headings
- [x] Add copy functionality to code blocks

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
|    | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|✓  | :scroll: Docs              |

### CODE REVIEW

File: docs/organizations/metadata.mdx [hunk 1]
```diff
@@ -1,5 +1,92 @@
 ---
-sanity_slug: /docs/organizations/organization-metadata
+title: Organization Metadata
+description: >-
+  Organization objects hold a set of metadata that can be used internally to store arbitrary information.
```
Comment (alexisintech) [inline docs/organizations/metadata.mdx:4]:
```suggestion
description: Organization objects hold a set of metadata that can be used internally to store arbitrary information.
```

File: docs/organizations/metadata.mdx [hunk 2]
```diff
@@ -1,5 +1,92 @@
...
 ---
 
-# This is a stub
+import { Tables } from "@/components/Table";
```
Comment (alexisintech) [inline docs/organizations/metadata.mdx:7]: Imports are unnecessary! The docs get rendered by our marketing repo, which is where the components live - they get rendered by the marketing repo.

```suggestion
import { Tables } from "@/components/Table";
```

File: docs/organizations/metadata.mdx [hunk 3]
```diff
@@ -1,5 +1,92 @@
...
+
+# Organization Metadata
+
+**Organization objects hold a set of metadata that can be used internally to store arbitrary information.**
+
+## Overview
+
+Metadata allows for custom data to be saved on the Organization object, Organization Invitation object, and Organization Membership object. There are two types of metadata: "**public**", and "**private**". Choosing which metadata field you should use for setting custom attributes on your organizations becomes a matter of access and visibility.
```
Comment (alexisintech) [inline docs/organizations/metadata.mdx:15]:
```suggestion
Metadata allows for custom data to be saved on the `Organization` object, `Organization Invitation` object, and `Organization Membership` object. There are two types of metadata: **public** and **private**. Choosing which metadata field you should use for setting custom attributes on your organizations becomes a matter of access and visibility.
```

File: docs/organizations/metadata.mdx [hunk 4]
```diff
@@ -1,5 +1,92 @@
...
+
+<Tables
+  headings={["Metadata", "Frontend API", "Backend API"]}
+  headingsMeta={[{maxWidth:'200px'},{maxWidth: '200px'},{maxWidth: '200px'}]}
```
Comment (alexisintech) [inline docs/organizations/metadata.mdx:19]: headingsMeta is a deprecated prop. we handle styling of components internally.

```suggestion
```

File: docs/organizations/metadata.mdx [hunk 5]
```diff
@@ -1,5 +1,92 @@
...
+
+## Backend metadata
+
+Both public and private metadata are set from the [Backend API](https://clerk.com/docs/reference/backend-api), but public metadata can be accessed from the [Frontend API](https://reference.clerk.dev/reference/frontend-api-reference) and [Backend API](https://clerk.com/docs/reference/backend-api). These types of metadata can be used to access non-sensitive information about an organization, such as their profile picture or their name. They can be used to build UIs where an organization might not be signed in, but you still want to show some of their profile info.
+
+## Organization Metadata
+
+### Public metadata
+
+The **publicMetadata** property should be used if you need to set some metadata from your back-end and have them displayed as read-only on the front-end.
```
Comment (alexisintech) [inline docs/organizations/metadata.mdx:54]: This is going to be implemented moving forward, but we will use backend and frontend without dashes.

```suggestion
The `publicMetadata` property should be used if you need to set some metadata from your backend and have them displayed as read-only on the frontend.
```

File: docs/organizations/metadata.mdx [hunk 6]
```diff
@@ -1,5 +1,92 @@
...
+
+```ts copy
```
Comment (alexisintech) [inline docs/organizations/metadata.mdx:56]: copy is used if you want to include a filename prop

```suggestion
```ts
```

File: docs/organizations/metadata.mdx [hunk 7]
```diff
@@ -1,5 +1,92 @@
...
+const updateOrg = await organizations.updateOrganizationMetadata(OrgID,{publicMetadata})
+```
+
+### Private metadata
+
+The **privateMetadata** property should be used if custom attributes contain sensitive information that should not be displayed on the front-end.
```
Comment (alexisintech) [inline docs/organizations/metadata.mdx:62]:
```suggestion
The `privateMetadata` property should be used if custom attributes contain sensitive information that should not be displayed on the frontend.
```

File: docs/organizations/metadata.mdx [hunk 8]
```diff
@@ -1,5 +1,92 @@
...
+
+```ts copy
```
Comment (alexisintech) [inline docs/organizations/metadata.mdx:64]:
```suggestion
```ts
```

File: docs/organizations/metadata.mdx [hunk 9]
```diff
@@ -1,5 +1,92 @@
...
+
+<Callout type="info" emoji="ℹ️">
+    Please note that Organization Invitations can only include **public** metadata.
+</Callout>
+
+<Callout type="warning">
+    Metadata is limited to **8kb** maximum.
+</Callout>
+
+## Backend metadata
```
Comment (alexisintech) [inline docs/organizations/metadata.mdx:46]: Let's remove this heading, I'm not convinced it's even necessary.

```suggestion
```

File: docs/organizations/metadata.mdx [hunk 10]
```diff
@@ -1,5 +1,92 @@
...
+
+```ts copy
+const updateOrg = await organizations.updateOrganizationMetadata(OrgID,{privateMetadata})
+```
+
+## Organization Membership Metadata
+
+### Public metadata
+
+The **publicMetadata** property should be used if you need to set some metadata from your back-end and have them displayed as read-only on the front-end.
```
Comment (alexisintech) [inline docs/organizations/metadata.mdx:72]:
```suggestion
The `publicMetadata` property should be used if you need to set some metadata from your back-end and have them displayed as read-only on the frontend.
```

File: docs/organizations/metadata.mdx [hunk 11]
```diff
@@ -1,5 +1,92 @@
...
+
+### Private metadata
+
+The **privateMetadata** property should be used if custom attributes contain sensitive information that should not be displayed on the front-end.
```
Comment (alexisintech) [inline docs/organizations/metadata.mdx:76]:
```suggestion
The `privateMetadata` property should be used if custom attributes contain sensitive information that should not be displayed on the frontend.
```

File: docs/organizations/metadata.mdx [hunk 12]
```diff
@@ -1,5 +1,92 @@
...
+
+```ts copy
+const updateOrg = await organizations.updateOrganizationMembershipMetadata(OrgID,{privateMetadata})
+```
+
+## Organization Invitation Metadata
+
+### Public metadata
+
+The **publicMetadata** property should be used if you need to set some metadata from your back-end and have them displayed as read-only on the front-end.
```
Comment (alexisintech) [inline docs/organizations/metadata.mdx:86]:
```suggestion
The `publicMetadata` property should be used if you need to set some metadata from your back-end and have them displayed as read-only on the frontend.
```

**Review by alexisintech — CHANGES_REQUESTED** [review]

**Comment by yiremorlans** [comment]
I appreciate the suggestions and thoroughly looking over the PR! Let me know if everything is in order.

**Review by alexisintech — APPROVED** [review]
Awesome! I'll be working on this page afterwards to update the content.
Thank you so much for the contribution! 😸💖

**Comment by alexisintech** [comment]
For reference, the docs before this PR:

https://github.com/clerkinc/clerk-docs/assets/98043211/c3f8e8bc-fc43-40b7-9ade-cc369ddf6133

**Pull request merged**